src/zui/ZUIButtonGroup/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { ButtonGroup } from '@mui/material';
+import { FC, ReactNode } from 'react';
+
+interface ZUIButtonGroupProps {
+  children: ReactNode;
```

Comment:
This allows for arbitrary children, which IMO is not great. Maybe the buttons should be defined as properties instead? Happy to discuss!